src/core/memory.d

```diff
@@ -141,12 +141,12 @@ private
     extern (C) GC.Stats gc_stats ( ) nothrow @nogc;
     extern (C) GC.ProfileStats gc_profileStats ( ) nothrow @nogc @safe;
```

Comment:
Everything except the following should not accept `scope` arguments. Especially the `add(Root|Range)` as their sole purpose is save (i.e. escape) the pointer argument.

Can accept `scope`:
* `(get|set|clr)Attr`
* `sizeOf`
* `query`

Reply:
Addressed with https://github.com/dlang/druntime/pull/2706

It appears the original code was wrong.  #2706 should set things straight.